rest-api/model/biobank_order.py

```diff
@@ -50,6 +54,12 @@ class BiobankOrder(Base):
   # in the resulting JSON.
   finalizedUsername = Column('finalized_username', String(255))
 
+  # cancelled finalized order may still be shipped to biobank for destruction
+  orderStatus = Column('order_status', Enum(BiobankOrderStatus))
+  cancelledTime = Column('cancelled_time', UTCDateTime)
```

Comment:
Change to last_modified. This should get updated whenever we modify an order in place.

Reply:
done